html/input/demos/inputs/index.html

```diff
@@ -158,42 +164,58 @@
     .input-label, .radio-set, .radio-legend {
       margin-bottom: 25px;
     }
+
+
+    @media screen and (max-width: 900px) {
```

Comment:
```suggestion
    @media screen and (max-width: 768px) {
```

Предлагаю взять такой же брейкпоинт, как на всём сайте.